spec/controllers/users_controller_spec.rb

```diff
@@ -21,12 +21,32 @@
       it "redirects to the sign in page" do
         expect(response).to redirect_to sign_in_path
       end
-    end
 
-    context "email sending" do
+      it "makes the user follow the inviter" do
+        luke = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: luke, recipient_email: 'hans@falcon.com')
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.